### DISCUSSION TITLE
Do I have to configure a chat widget on D365 to be able to use this sdk?

### DISCUSSION THREAD

**Discussion by linkinleon** [microsoft/omnichannel-chat-sdk#196] [Q&A] [answered]
Hi team,
    I would like to know if I have to configure a widget on D365 to use this SDK, can I use my own webchat widget and only process agent handover through this SDK?

**Comment by xTEddie** [comment, marked as answer]
@linkinleon Yes, a live chat widget needs to be configured on D365 portal before being able to use the SDK. 

Documentation: https://learn.microsoft.com/en-us/dynamics365/customer-service/add-chat-widget?tabs=customerserviceadmincenter
